client/src/components/Info.style.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  height: 100vh;
+  align-items: center;
+  flex-flow: column wrap;
+  justify-content: center;
+`;
+
+export const H1 = styled.h1`
+  font-weight: bold;
+  font-size: 3rem;
+  background-color: var(--highlight);
+  margin-top: 5%;
+`;
+
+export const P = styled.p`
+  font-style: var(--black);
+  width: 20%;
```

Comment:
Careful with this - it might be better to make this closer to 100% of the width, and then reduce the size of the ```main``` tag on bigger screens - that can be done in another issue though. If you view the page on mobile, this makes the flex display differently, and the text only takes up a tiny section of the page.